CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Changed
 
+- Renamed the root field to ca_certs and updated it to support multiple CA
+  certificates.
```

Comment:
I think this line needs to be combined with this line https://github.com/aicers/crusher/pull/94/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR33
```
- Changed configuration fields name.
  - `roots` to `root`.
```

So I would like to suggest removing L17-18, and instead modifying L33 to

``` 
  - `roots` to `ca_certs`. It also introduces support for multiple CA certificates.
  certificates.
```

Reply:
I removed the first line and combined with that line as you pointed out.